codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -1249,11 +1421,15 @@ impl GqlScalar {
             GqlScalar::ID => PGPrimitive::Text,
             GqlScalar::String => PGPrimitive::Text,
             GqlScalar::Int => PGPrimitive::Integer,
-            GqlScalar::Float => PGPrimitive::Numeric, // Should we allow this type? Rounding issues will abound.
+            GqlScalar::Float => PGPrimitive::Numeric(None), // Should we allow this type? Rounding issues will abound.
```

Comment:
I didn't want to get sidetracked on this and I'd already spent a lot of time, but maybe for float we should be setting both a precision and a scale since it is just a javascript float. But I think javascript floats can behave quite weirdly and used in very different flexible ways. The main thing is that they lose precision, so probably just leaving the default is fine.